server/db/db.js

```diff
@@ -245,6 +245,7 @@ async function getDataForExport(pgClient) {
         s.chatbot_state AS "Session State",
         s.alert_type As "Alert Type",
         s.responded_by_phone_number AS "Session Responded By",
+        s.number_of_alerts AS "Number Of Alerts",
```

Comment:
Good points! OK, for simplicity's sake and to not change Oona's flow working with this data, I think I'll just not add the column. If it becomes interesting it can be added later. :)